src/v/pandaproxy/client/record_set.cc

```diff
@@ -0,0 +1,82 @@
+// Copyright 2020 Vectorized, Inc.
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.md
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0
+
+#include "pandaproxy/client/record_set.h"
+
+#include "kafka/requests/kafka_batch_adapter.h"
+
+namespace pandaproxy::client {
+
+struct mini_header {
```

Comment:
mini header for what? can we give this a better name.

Reply:
is this `pandaproxy::client::internal`?

Reply:
Fixed in #515